shm/Player.hpp

```diff
@@ -4,19 +4,29 @@
 #include "Cargo.hpp"
 #include <memory>
 
+// Class responsible for managing player in the game world. Should be only one instance (for now).
 class Player {
-    std::unique_ptr<Ship> ship_;
-    size_t money_;
-    size_t availableSpace_;
-
-    void calculateAvailableSpace();
-
 public:
     Player(std::unique_ptr<Ship> ship, const size_t& money);
-    const std::unique_ptr<Ship>& getShip() const;
+
+    // Returns constant raw pointer to ship in players possesion if one exists, otherwise returns nullptr.
+    const Ship* getShip() const;
+
+    // Returns constant raw pointer to cargo instance of index in cargo vector provided as a parameter.
+    // If cargo object doesn't exist - returns nullptr
+    const Cargo* getCargo(size_t index) const;
+
+    // If player is in a possesion of a ship this method prints out all the cargo from the ship to a screen.
+    void printCargoManifest() const;
+
     size_t getMoney() const;
     size_t getAvailableSpace() const;
     size_t getSpeed() const;
-    Cargo* getCargo(size_t index) const;
-    void printCargoManifest() const;
+
+private:
+    std::unique_ptr<Ship> ship_;
+    size_t money_;
+    size_t availableSpace_;
+
+    void calculateAvailableSpace();
```

Comment:
This is my preference, so feel free to ignore this comment.

Class members should always be at the bottom. So move calculateAvailableSpace() a the top.